proxy/src/main/java/com/wavefront/agent/logsharvesting/FilebeatListener.java

```diff
@@ -85,6 +83,20 @@ public FilebeatListener(PointHandler pointHandler, LogsIngestionConfig logsInges
           }
         })
         .build(new MetricCacheLoader(this.metricsRegistry));
+
+    flushTimer = new Timer();
+    long flushMillis = TimeUnit.SECONDS.toMillis(logsIngestionConfig.aggregationIntervalSeconds);
+    flushTimer.schedule(new TimerTask() {
+      @Override
+      public void run() {
+        try {
+          flush();
+        } catch (Exception e) {
+          logger.severe("Error flushing: " + e.getMessage());
```

Comment:
logger.log(Level.SEVERE, "Uncaught exception when flushing metrics", e);

and remove the e.printStackTrace()